### ISSUE TITLE
Failed to parse source map

### ISSUE THREAD

**Issue by braddi** [gregjacobs/Autolinker.js#396]
I have upgraded to CRA 5 and I'm getting this compilation error:

```
WARNING in ./node_modules/autolinker/dist/es2015/truncate/truncate-end.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '\node_modules\autolinker\src\truncate\truncate-end.ts' file: Error: ENOENT: no such file or directory, open '\node_modules\autolinker\src\truncate\truncate-end.ts'
```

![image](https://user-images.githubusercontent.com/100603994/208423084-a2c0918f-ef47-4de5-8750-9dd64b0d1c8e.png)

```
      "dependencies": {
        "autolinker": {
          "version": "3.16.2",
          "resolved": "https://registry.npmjs.org/autolinker/-/autolinker-3.16.2.tgz",
          "integrity": "sha512-JiYl7j2Z19F9NdTmirENSUUIIL/9MytEWtmzhfmsKPCp9E+G35Y0UNCMoM9tFigxT59qSc8Ml2dlZXOCVTYwuA==",
          "requires": {
            "tslib": "^2.3.0"
          }
        },
```
Somehow CRA keeps trying to find the map files in src folder instead of dist.

**Comment by yingxinj** [comment]
+1, I have the same error. I believe that we could either set `sourcemap: false` in rollup.config.ts or include `files: src` in `package.json`, but I'm not sure which one would be preferable?

**Comment by k-funk** [comment]
Also running into this issue via installing https://www.npmjs.com/package/swagger-ui-react, which depends on https://www.npmjs.com/package/remarkable, which depends on Autolinker 3.x.

**Comment by gregjacobs** [comment]
Apologies on the delay on this. Source maps should be fixed with Autolinker 4.0.2+
